### PR TITLE
Merge reproducibility docs into main

### LINKED ISSUE #605: Release the reproducibility docs
This issue tracks merging `reproducibility-docs` into `main` to finish out #487

### PR DESCRIPTION
**Pull request by sjspielman**
Closes #605 

This PR releases the reproducibility docs. I merged `main` into `reproducibility-docs` and fixed conflicts to get all the parallel changes first, and now sending everything back into `main`. Happy diff-ing 😬 

Edit: Looks like given all the many file changes here, all the (non-me) code owners were automatically requested for review. Having multiple eyes on this is probably a very good idea though!!

### CODE REVIEW

File: analyses/cell-type-ewings/results/README.md
Comment (jashapiro) [inline analyses/cell-type-ewings/results/README.md:1]: I don't know that we want changes in this module merged? @allyhawkins

Comment (allyhawkins) [reply analyses/cell-type-ewings/results/README.md:1]: No we definitely don't...

Comment (sjspielman) [reply analyses/cell-type-ewings/results/README.md:1]: Hmm probably I should not have merged `main` in here...probably calls for a fresh PR?

**Review by allyhawkins — COMMENTED** [review]
This looks fine to me other than any of the files that are being changed in the Ewing's module. All of those changes should not be there.

**Comment by sjspielman** [comment]
Nope, I did too much, starting from scratch.

**Pull request closed without merging**